packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -50,6 +55,11 @@
 			height: 30px;
```

Comment:
Can we remove this height. It is not needed as the min height works fine. The height is what is making the clipping issue. 
```suggestion

```